src/vhdljjparser.cpp

```diff
@@ -397,14 +397,7 @@ void VHDLOutlineParser::handleCommentBlock(const char *doc1, bool brief)
     s->current->docLine = p->yyLineNr;
   }
 
-  int j = doc.find("[plant]");
-  if (j >= 0)
-  {
-    doc = doc.remove(j, 7);
-    s->current->stat = true;
-  }
-
```

Comment:
I think the new implementation is better, though 2 remarks:
- consistency, although `[plant]` is not documented at least I couldn't find it, might be useful to al least still strip the `[plant]`
- not for this pull request but for future new implementations: doxygen uses `{...}` for options on commands, see e.g. the `\snippet` command